### DISCUSSION TITLE
Comments and Tagging Blogs

### DISCUSSION THREAD

**Discussion by iamjatindersingh** [thuliteio/doks#425] [Ideas]
I have been using DOKS for the last few weeks and extremely impressed with it and its configuration. I am opening up this discussion to suggest if you could add comments` (3rd party integration examples) for the blog posts and out-of-the-box tagging/grouping for blogs. And it would be great to then have the tags searchable via flex-search.

Thanks.

**Comment by ddomdel** [comment]
Yes like being able to use [giscus ](https://giscus.app/) as well.

@iamjatindersingh Have you tried experimenting with Disqus? It seems that it is available in Hugo. https://gohugo.io/content-management/comments/

**Comment by h-enk** [comment]
Tags is planned for with the next release — see also https://github.com/h-enk/doks/issues/501

Adding a comments system is more for a guide (https://getdoks.org/docs/how-to/external-features/)
